--- ostree-2019.6/autogen.sh
+++ ostree-2019.6.patch/autogen.sh
@@ -39,7 +39,7 @@
 # FIXME - figure out how to get aclocal to find this by default
 ln -sf ../libglnx/libglnx.m4 buildutil/libglnx.m4
 
-autoreconf --force --install --verbose
+autoreconf --force --install --verbose --include=/home/andree/dev/linux/Andreelec.tv/build.custom-RPi2.arm-9.80-devel/toolchain/armv7ve-libreelec-linux-gnueabi/sysroot/usr/share/aclocal
 
 cd $olddir
 test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
